extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/AstroHandlerFactory.java

```diff
@@ -21,6 +20,7 @@
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 
+import com.google.common.collect.Maps;
```

Comment:
FYI: See https://github.com/eclipse/smarthome/issues/2801 that we prefer to use Guava as little as possible and rather want to get rid off it completely one day. So unless tremendously helpful, better choose another option.